uk.me.doof.Lander.json

```diff
@@ -0,0 +1,29 @@
+{
+    "app-id": "uk.me.doof.Lander",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "lander",
+    "finish-args": [
+        "--socket=x11",
```

Comment:
you need --share=ipc as well if you have x11

Reply:
Done, thanks.